src/app/functions/server/onerep.ts

```diff
@@ -361,7 +361,7 @@ export async function isEligibleForFreeScan(
   }
 
   if (!user?.subscriber?.id) {
-    throw new Error("No session");
+    throw new Error("No fxa_uid found in session");
```

Comment:
This error message looks wrong now, as this is checking for the ID of the user in our own DB table, not the ID of the FxA user.

Reply:
Changed myself since you asked me to merge this: 4eebbaedd.